### PR TITLE
support delete -f

### PR DESCRIPTION
**Pull request by Stevent-fei**

### CODE REVIEW

**Review by VinceCui — COMMENTED** [review]
sealer delete -a should be retain.

File: cmd/sealer/cmd/cluster/delete.go [hunk 1]
```diff
@@ -85,8 +83,9 @@ func NewDeleteCmd() *cobra.Command {
 	deleteFlags = &types.Flags{}
 	deleteCmd.Flags().StringVarP(&deleteFlags.Masters, "masters", "m", "", "reduce Count or IPList to masters")
 	deleteCmd.Flags().StringVarP(&deleteFlags.Nodes, "nodes", "n", "", "reduce Count or IPList to nodes")
+	deleteCmd.Flags().StringVarP(&deleteClusterFile, "Clusterfile", "f", "", "delete a kubernetes cluster with Clusterfile")
```
Comment (VinceCui) [inline cmd/sealer/cmd/cluster/delete.go:86]: default is common.GetDefaultClusterfile()

**Comment by Stevent-fei** [comment]
> 

sealer delete -a this function has not been deleted

File: cmd/sealer/cmd/cluster/delete.go (hunk 1 above)
Comment (Stevent-fei) [reply cmd/sealer/cmd/cluster/delete.go:86]: > 

"sealer delete -a" deletes files under ".sealer/Clusterfile" by default

If you specify the default delete path, "sealer delete -f" loses the meaning of the specified file

**Review by kakaZhou719 — APPROVED** [review]

**Pull request merged**